indico/modules/events/registration/controllers/management/fields.py

```diff
@@ -317,6 +320,7 @@ def _process(self):
         field_data = snakify_keys(request.json['fieldData'])
         form_field = RegistrationFormField(parent=self.section, registration_form=self.regform)
         _fill_form_field_with_data(form_field, field_data)
+        signals.event.registration_form_field_added.send(form_field)
```

Comment:
Do this after the flush so any plugin handling the signal has a fully-flushed object that's in the DB